server/schemas/mutations/delete-expense.mutation.js

```diff
@@ -0,0 +1,24 @@
+import {
+  GraphQLString,
+  GraphQLNonNull
+} from 'graphql';
+
+import Expense from '../../models/expense.model';
+
+export default {
+  type: GraphQLString,
+  args: {
+    expenseId: { type: new GraphQLNonNull(GraphQLString) },
+  },
+  resolve: async (obj, { expenseId }) => {
+    try {
+      const deleted = await Expense.findByIdAndRemove(expenseId).exec();
+      if (deleted) {
+        return 'Successfully deleted!';
+      }
+      throw new Error('Unable to delete');
```

Comment:
Hmmm. I see what you are trying to do, however, the error might be handled in the catch block. No?

Have you tested to find that you get the 'Unable to delete' message?

Reply:
Yep. I've been able to verify this locally. 

Basically, it throws this error when you try a `findById` operation for an ID that does not exist in the DB. MongoDB (or Mongoose) returns `null`. I don't want it to return `null` so I make it throw this error @oreoluwade

Reply:
Awesome!